test/server/routes/findbyid.js

```diff
@@ -0,0 +1,119 @@
+const findbyid = require('../../../server/routes/findbyid');
+const _ = require('lodash');
+const identity = () => {};
+
+const makeRequest = opts => {
+  const req = {};
+  _.set(req, 'app.locals.ph.store.getMany', opts.getMany);
+  _.set(req, 'query.ids', opts.ids);
+  _.set(req, 'query.lang', opts.lang);
+  return req;
+};
+
+const makeResponse = opts => {
+  return {
+    status: status => {
+      if (opts.status) { opts.status(status); }
+      return { send: opts.send || identity, json: opts.json || identity };
+    }
+  };
+};
+
+module.exports.all = (test, common) => {
+  test('parse ids - correct numbers list with spaces', t => {
+    const req = makeRequest({
+      ids: '85682555, 85633111,102064231 , 85682523 ,    102063845    ,',
+      getMany: function(ids) {
+        t.deepEqual(ids, [85682555, 85633111, 102064231, 85682523, 102063845]);
+        t.end();
+      }
+    });
+    findbyid(req, null);
+  });
+
+  test('parse ids - incorrect numbers', t => {
+    const req = makeRequest({
+      ids: 'not a number, 85633111a,1d02064231',
+      getMany: function(ids) {
+        t.deepEqual(ids, [85633111, 1]);
```

Comment:
Is this really expected?

Reply:
hmm.. it seems that `parseInt('1d02064231',10) === 1`

it's not very elegant, I guess we can either stick with this or just ignore any ids which contain non-numeric characters.

Reply:
Yes parseInt stops when there is a non numeric character.
Ok, so I think we should leave this like that, this will avoid useless checks.

Reply:
:+1: